.github/workflows/ci.yml

```diff
@@ -423,24 +425,35 @@ jobs:
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-6-non-generic
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-7-non-generic
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-8-non-generic
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-tpcds
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-oauth2
+            jdk: 11
           # this suite is not meant to be run with different configs
           - config: default
             suite: suite-compatibility
+            jdk: 11
           # this suite is designed specifically for apache-hive3. TODO remove the suite once we can run all regular tests on apache-hive3.
           - config: apache-hive3
             suite: suite-hms-only
+            jdk: 11
+          # suite-1 does not contain Kerberos/encrypted Hive environments
+          - config: default
```

Comment:
would it make more sense to run in on hdp3, as it is more recent Hive distro (and enables somewhat more testcases)

Reply:
@losipiuk I can try that as a follow-up. I wanted to enable anything that passed on JDK17 so we can at least have a minimal coverage here.